src/Stratis.Bitcoin.IntegrationTests/RPC/RPCTestsMutable.cs

```diff
@@ -67,6 +68,62 @@ public void TestRpcGetBlockWithValidHashIsSuccessful()
             }
         }
 
+        [Fact]
+        public void TestRpcListUnspentWithDefaultsIsSuccessful()
+        {
+            using (NodeBuilder builder = NodeBuilder.Create(this))
+            {
+                Network network = new BitcoinRegTest();
+                var node = builder.CreateStratisPowNode(network).WithWallet().Start();
+
+                RPCClient rpcClient = node.CreateRPCClient();
+                const int numCoins = 5;
+                int blocksToMine = (int)network.Consensus.CoinbaseMaturity + numCoins;
+                TestHelper.MineBlocks(node, blocksToMine);
+                TestHelper.WaitLoop(() => node.FullNode.GetBlockStoreTip().Height == blocksToMine);
+
+                var coins = rpcClient.ListUnspent();
+                coins.Should().NotBeNull();
+                coins.Length.Should().Be(numCoins);
+            }
+        }
+
+        [Fact]
+        public void TestRpcListUnspentWithParametersIsSuccessful()
+        {
+            using (NodeBuilder builder = NodeBuilder.Create(this))
+            {
+                Network network = new BitcoinRegTest();
+                var node = builder.CreateStratisPowNode(network).WithWallet().Start();
+
+                RPCClient rpcClient = node.CreateRPCClient();
+                const int numCoins = 5;
+                int blocksToMine = (int)network.Consensus.CoinbaseMaturity + numCoins;
+                TestHelper.MineBlocks(node, blocksToMine);
+                TestHelper.WaitLoop(() => node.FullNode.GetBlockStoreTip().Height == blocksToMine);
+
+                var minerAddress = BitcoinAddress.Create(node.MinerHDAddress.Address, network);
+
+                // validate existing address and minconf
+                var coins = rpcClient.ListUnspent((int)network.Consensus.CoinbaseMaturity+2, 99999, minerAddress);
```

Comment:
Doesn't this method take an array of `BitcoinAddress`?

Reply:
```
        public UnspentCoin[] ListUnspent(int minconf, int maxconf, params BitcoinAddress[] addresses)
        {
            IEnumerable<string> addr = from a in addresses select a.ToString();
            RPCResponse response = SendCommand(RPCOperations.listunspent, minconf, maxconf, addr.ToArray());
            return response.Result.Select(i => new UnspentCoin((JObject)i, this.Network)).ToArray();
        }
```